docker-compose.yml

```diff
@@ -0,0 +1,55 @@
+version: "3.8"
+
+services:
+
+  db:
+    container_name: dds_database
+    image: mariadb:latest
+    environment:
+      - MYSQL_ROOT_PASSWORD=${DDS_MYSQL_ROOT_PASS}
+      - MYSQL_USER=${DDS_MYSQL_USER}
+      - MYSQL_PASSWORD=${DDS_MYSQL_PASS}
+      - MYSQL_DATABASE=DeliverySystem
+    restart: on-failure
+    ports:
+      - 127.0.0.1:3306:3306
+    # Test to see if the database is accessible. Backend waits for this with 'condition: service_healthy'
+    healthcheck:
+      test: mysqladmin ping -h 127.0.0.1 -u $$MYSQL_USER --password=$$MYSQL_PASSWORD
+      interval: 10s
+      timeout: 5s
+      retries: 5
+
+  backend:
+    container_name: dds_backend
+    image: dds-backend-local:latest
+    build: ./
+    command: python3 app.py
+    environment:
+      - DDS_APP_CONFIG
+      - FLASK_ENV
```

Comment:
Here can you also add `DDS_CLI_ENV`, atm its needed

Reply:
That's already in the `.env` file, set as `development` by default.